### PR TITLE
added blog post section to index

### PR DESCRIPTION
**Pull request by SingingApple**
Blog post fetches data from a dummy JSON file using blog.js file. Blogs currently display truncated article body, article title, article source, author's name and published date in a card on the landing page. For date formatting, momentjs has been used. 
Two .png files have also been added for logos under /img/icons/blog. (medium and hackernoon)

### CODE REVIEW

**Comment by rajat2502** [comment]
@krrohitch Sir please review the design of the blog section.

**Comment by rajat2502** [comment]
> momentjs

Don't use any external libraries until it is too necessary.

**Comment by SingingApple** [comment]
Added img to each card and aligned all items. Please review

File: index.html [hunk 1]
```diff
@@ -2,231 +2,417 @@
...
+            id="top-button"
+            class="smoothScroll"
+            title="Back to Top"
+          >
+            <span class="fa fa-chevron-circle-up fa-2x"></span>
+          </a>
+        </div>
+      </section>
+    </div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.26.0/moment.min.js"></script>
```
Comment (rajat2502) [inline index.html]: Remove this line.

File: js/Blog.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "posts": [
+    {
+      "title": "Hello World",
+      "body": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi, quibusdam. Officiis veritatis quis praesentium natus distinctio molestias magni reprehenderit sapiente!",
+      "author": "John Doe",
+      "published_on": "20 November 2019",
+      "source": "medium",
+      "link": "https://medium.com",
+      "image": "../img/zealicon.png"
```
Comment (rajat2502) [inline js/Blog.json]: For now, this is fine but later we will have hosted image links here

Comment (rajat2502) [reply js/Blog.json]: Also, don't save the blog website image in the images folder we can also add their link here.

File: js/blog.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+//Async request to JSON file for Blog post
+const getPosts = async () => {
+  const response = await fetch("../js/Blog.json");
+  const data = response.json();
+  return data;
+};
```
Comment (rajat2502) [inline js/blog.js]: We don't need to make an API call to the blog's file. You can directly import the object from the .json file.

**Review by rajat2502 — CHANGES_REQUESTED** [review]

File: js/Blog.json (hunk 2 above)
Comment (SingingApple) [reply js/Blog.json]: yes sir, links will also work as [image] value is being rendered for background URL.

**Comment by SingingApple** [comment]
- Changed card body font to Avenir regular
- Added hyperlink to card container
- changed background-size for card image
- decreased y-offset for box-shadow of cards
- changed fetch API 
@krrohitch @rajat2502  please review

**Comment by rajat2502** [comment]
@rex12543 Please review it design-wise.
https://deploy-preview-38--trusting-mayer-c6cce8.netlify.app

**Comment by rex12543** [comment]
> @rex12543 Please review it design-wise.
> https://deploy-preview-38--trusting-mayer-c6cce8.netlify.app

The blog cards can be scaled down in size.
Rest looks fine to me for now. ✌

**Comment by rajat2502** [comment]
@SingingApple, update this PR according to the new design. Also, make sure that you resolve any merge conflicts.

**Comment by SingingApple** [comment]
@rajat2502 please review!

**Pull request closed without merging**